app/Command/Docs.hs

```diff
@@ -60,19 +55,17 @@ docgen (PSCDocsOptions fmt inputGlob output) = do
     hPutStrLn stderr "purs docs: no input files."
     exitFailure
 
+  ms <- parseAndConvert input
   case fmt of
-    Etags -> dumpTags input dumpEtags
-    Ctags -> dumpTags input dumpCtags
+    Etags -> mapM_ putStrLn $ dumpEtags $ zip input ms
```

Comment:
I don’t think it’s safe to assume that the modules come out in the same order that they went in. We might need to modify `parseAndConvert` (instead of doing a `zip` here) so that it gives you the file path associated with each module.

Reply:
I'll take a look at this, thanks.